### PR TITLE
feat: support tsma

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: src/benchInsert.c [hunk 1]
```diff
@@ -1570,6 +1570,69 @@ static int startMultiThreadInsertData(SDataBase* database, SSuperTable* stbInfo)
...
+    int code = queryDbExec(conn, command);
+    if (code == 0) {
+        infoPrint(stdout, "successfully create tsma with command <%s>\n", command);
+    }
+}
+
+static void* create_tsmas(void* args) {
+    tsmaThreadInfo* pThreadInfo = (tsmaThreadInfo*) args;
+    int inserted_rows = 0;
+    SBenchConn* conn = init_bench_conn();
```
Comment (sangshuduo) [inline src/benchInsert.c:1611]: is it possible to get a NULL pointer here?

File: src/benchInsert.c [hunk 2]
```diff
@@ -1630,6 +1693,23 @@ int insertTestProcess() {
             }
         }
     }
+    
+    if (g_arguments->taosc_version == 3) {
+        for (int i = 0; i < g_arguments->databases->size; i++) {
+            SDataBase* database = benchArrayGet(g_arguments->databases, i);
+            for (int j = 0; j < database->superTbls->size; ++j) {
+                SSuperTable* stbInfo = benchArrayGet(database->superTbls, j);
```
Comment (sangshuduo) [inline src/benchInsert.c:1701]: is it possible to get a NULL pointer here?

**Review by sangshuduo — COMMENTED** [review]
see my comments

File: src/benchInsert.c (hunk 1 above)
Comment (zhaoyanggh) [reply src/benchInsert.c:1611]: yea, it could be, will fixed

**Review by afwerar — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**